worker/docker_client.py

```diff
@@ -409,8 +345,17 @@ def check_finished(self, container_id):
             
             inspect_json = json.loads(inspect_response.read())
             if not inspect_json['State']['Running']:
-                return (True, inspect_json['State']['ExitCode'], None)
-            return (False, None, None)   
+                # If the logs are nonempty, then something might have gone
+                # wrong with the commands run before the user command,
+                # such as bash or cd.
```

Comment:
Is it feasible to instead check the exitcode rather than checking if logs are empty? https://docs.docker.com/engine/reference/run/#/exit-status